docs/scarpet/api/Auxiliary.md

```diff
@@ -142,6 +142,33 @@ Available shapes:
      * `height` - height of the cyllinder, defaults to `0`, so flat disk. Can be negative.
      * `level` - level of details, see `'sphere'`.
 
+ * `'poly'`:
+   * Required attributes:
+     * `points` - list of points defining vertices of the polygon
+   * Optional attributes:
+     * `relative` - list of bools. vertices of the polygon that affected by 'follow'. Default means that every point is affacted.
+     * `mode` - how those points are connected. may be "polygon"(default),"strip" or "triangles". "polygon" means that it will be viewed as vertices of a polygon center on the first one. "strip" means that it will be viewed as a triangles strip. "triangles" means that it will be viewed as some triangles that are not related to each other (therefor length of `points` in this mode have to be a multiple of 3).
+     * `inneredges` - if `true` it will make the inner edges be drawn as well. 
+     * `doublesided` - if `true` it will make the shapes visible from the back as well. Default is `true`. 
+     * `needf3b` - if True, can toggle visibility by using F3+B. just like entities' hitbox.
```

Comment:
this option sounds very useful, not just for these shapes but for any shape as a general option. But I would definitely change its name. If we want a shape to appear only when hitboxes option is toggled, then I would maybe call it 'toggleable', or 'debug' indicating it would be used with F3B debug mode. thoughts on name?

Reply:
ok. i will call it 'toggleable' instead.